config/logging.yml

```diff
@@ -27,14 +27,32 @@ log_cache_gateway:
 log_cache_syslog:
   crt: #@ data.values.log_cache_syslog.crt
   key: #@ data.values.log_cache_syslog.key
+
+log_cache_external:
+  crt: #@ data.values.log_cache_external.crt
+  key: #@ data.values.log_cache_external.key
+
+client:
+  id: #@ data.values.log_cache_client.id
+  secret: #@ data.values.log_cache_client.secret
 #@ end
 
 #@ logging = library.get("github.com/cloudfoundry/cf-k8s-logging")
 --- #@ template.replace(logging.with_data_values(logging_values()).eval())
 
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: system-ca
```

Comment:
`system-ca` is way too of a generic name for a secret. i would recommend to either make secret name configuration via one of the data values, or hard coding to something more apparent like log-cache-system-ca.

Reply:
This *isn't* the log-cache-system ca, it's the ca used to sign both uaa and capi. I hear there might be a future where uaa and capi have their own ca/certs, I'm open to changing it then.

Reply:
We could make the secret name configurable though, that's possibly a good move.

Reply:
> This isn't the log-cache-system ca

it's a ca that log-cache consumes though, hence i think making it more "injectable" is better (ie yall had to add it as a Secret resource.)

>  I hear there might be a future where uaa and capi have their own ca/certs

longer term (which hopefully happens soon) we ll be relying on istio to do all tls wrapping/unwrapping so there wouldnt be CAs to configure between components (ie no system ca)